### ISSUE TITLE
Sony package addition

### ISSUE THREAD

**Issue by lawson85** [0x192/Universal-Android-Debloater#584]
**Your phone:** Sony Xperia X F5121

## Document each package the best you can
**List**: `Google`|`Misc`|`Oem` (manufacturer)|`Aosp`|`Pending`|`Carrier` (isp).
**Removal**: `Recommended`, `Advanced`, `Expert` (this can break important features), `Unsafe` (this can bootloop the phone or break extremely important features).

### \com.sony.tvsideview.phone\
**List**: \oem\ 
**Removal**: \<recommendation list\>
> Description. Sony's TV remote control app Video＆TV SideView will make your TV viewing experience more enjoyable.\nhttps://play.google.com/store/apps/details?id=com.sony.tvsideview.phone

### \com.scee.psxandroid\
**List**: \<oem\> 
**Removal**: \<recommendation list\>
> Description. Stay connected to your gaming friends and the games you love to play, wherever you go with PlayStation App. See who's online, voice chat and send messages, and discover deals on PS Store. https://play.google.com/store/apps/details?id=com.scee.psxandroid

### \<com.sonyericsson.android.conversations\>
**List**: \<oem\> 
**Removal**: \<expert\>
> Description. Sony's default messages (SMS) app. https://www.apkmirror.com/apk/sony-mobile-communications/messaging-sony-mobile-communications/messaging-sony-mobile-communications-29-3-a-6-12-release/messaging-29-3-a-6-12-android-apk-download

Some other Sony apps: https://www.apkmirror.com/uploads/?devcategory=sony-mobile-communications

**Comment by AnonymousWP** [comment]
Are you sure those first two packages are pre-installed on a phone? Why didn't you enter a phone model? It needs to be bloatware, not just any app you can download from the Play Store.

**Comment by lawson85** [comment]
Yes they are. I did add the phone model numbers but the angle brackets <> are confusing.

**Comment by AnonymousWP** [comment]
Alright, thanks. Fixed.
